html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form>
+    <div>Email</div>
+    <input type="text" name="email"/>
+  </form>
+
+  <form>
```

Comment:
You do not want to have individual `form` elements like this, since all of these inputs are related. You should surround all the inputs here with one `form` element.

Reply:
I see. Thank you! I just submitted my adjustment